src/components/common/SearchBar/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import searchIcon from "assets/images/search.svg";
+
+const SearchBar = ({ placeholder }) => {
```

Comment:
To not create a limitation in terms of which attributes you want to pass to the input you can simply do this.
```suggestion
const SearchBar = (...props) => {
```